linalg/hypre.hpp

```diff
@@ -739,6 +760,26 @@ class HypreSolver : public Solver
    virtual ~HypreSolver();
 };
 
+
+#if MFEM_HYPRE_VERSION >= 21800
+/// Abstract class for hypre's solvers and preconditioners
```

Comment:
Is this class used? Is it necessary?

Reply:
For less than ~500 processors, this is actually faster than AIR or ILU in solving Ex9p, but the example is pretty full in terms of different uses and solvers w/ ILU, AIR, PA, and explicit time stepping, so I did not add support for this.

Reply:
Thanks for clarifying. I am reluctant to ask for the PR to be extended to add support for this solver in ex9p, but if it is the fastest solver for less than ~500 processors, it seems very desirable. The examples are typically run with a small number of processors. @pazner would you want to see this added to ex9p?

Reply:
My guess is that the difference in speed is relatively marginal. Is that right @ben-s-southworth? I don't think we need to overload the example with too many options.

Reply:
@dylan-copeland I think the difference is not enough to warrant another solver type in this example. It is also somewhat specialized for this problem, but if a user tried to apply it to a different problem it may not work, whereas AIR/ILU are more likely to work in a variety of settings.